src/platform_impl/linux/wayland/keyboard.rs

```diff
@@ -305,7 +305,7 @@ fn keysym_to_vkey(keysym: u32) -> Option<VirtualKeyCode> {
         // misc
         // => Some(VirtualKeyCode::AbntC1),
         // => Some(VirtualKeyCode::AbntC2),
-        keysyms::XKB_KEY_plus => Some(VirtualKeyCode::Add),
+        // keysyms::XKB_KEY_plus => Some(VirtualKeyCode::Add),
```

Comment:
Why remove this? People might be relying on this, right? If anything, we should add another virtual keycode for these.

Reply:
Well, the input model in winit is unclear, and you can't add those on other platforms anyway(except X11), so I've decided to remove it to avoid further confusion.

Don't get me wrong, I've tried adding new codes initially, but it turned out that those will be only on X11/Wayland, so I've reverted it.

Reply:
Is having keycodes for just X11/Wayland a problem? That seems better than removing it completely.

Reply:
Kind of, since you're relying on those being accessed via shift + equals right now, so if you add special keys for those you change that implication, meaning that macOS/Windows/web not supporting those is a bug.

I can add those again that's not a big deal.

Reply:
Just doesn't sound great to remove features like that. Especially since people might be relying on it.

Reply:
Ok, I've added those.